crates/tuono/src/app.rs

```diff
@@ -199,6 +203,7 @@ impl App {
             .stdin(Stdio::piped())
             .stdout(Stdio::piped())
             .stderr(Stdio::piped())
+            .envs(std::env::vars())
```

Comment:
I'm not a fan of having the vars in both processes (CLI and server).
What about passing the var using a vector?

```suggestion
            .envs(self.env_var_manager.variables)
```

Reply:
Much better idea, will do that, thanks!

Reply:
That hash set `self.env_var_manager.variables` only contains the names. We store the actual values in `std::env`. Should we keep doing that? WDYT